app_test.go

```diff
@@ -245,6 +245,16 @@ func Test_App_ErrorHandler_RouteStack(t *testing.T) {
 	utils.AssertEqual(t, "1: USE error", string(body))
 }
 
+func Test_App_serverErrorHandle(t *testing.T) {
```

Comment:
Test_App_serverErrorHandler_Internal_Error  would be better i suppose

Reply:
+1 & done